remove torchao dependency in examples/llama2

manuelcandales opened the pull request:
Differential Revision: D55282262

guangy10 commented:
Need to enable trunk flow becasue we want to know if it actually resolve the macos jobs

manuelcandales committed:
remove torchao dependency in examples/llama2 (#2621)

Summary:

Removes torchao dependency in examples/llama2
Disables GPTQ path

Reviewed By: iseeyuan, guangy10

Differential Revision: D55282262
